sympy/geometry/ellipse.py

```diff
@@ -1436,7 +1437,11 @@ def __new__(cls, *args, **kwargs):
             elif len(args) == 2:
                 # Assume (center, radius) pair
                 c = Point(args[0], dim=2)
-                r = sympify(args[1])
+                r = Tuple(args[1], )
+                r = r.xreplace(dict(
+                [(f, simplify(nsimplify(f, rational=True)))
+                 for f in r.atoms(Float)]))
+                r = r[0]
```

Comment:
Can the lines above be simplified to `r = nsimplify(args[1], rational=True)`?

Reply:
I don't understand why you've created a `Tuple`...

Reply:
Thanks for the suggestion. I am changing it.